plugins/generic/externalFeed/controllers/grid/ExternalFeedGridHandler.inc.php

```diff
@@ -0,0 +1,240 @@
+<?php
+/**
+ * @file controllers/grid/ExternalFeedGridHandler.inc.php
+ *
+ * Copyright (c) 2014-2016 Simon Fraser University Library
+ * Copyright (c) 2003-2016 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @class ExternalFeedGridHandler
+ * @ingroup controllers_grid_externalFeed
+ *
+ * @brief Handle external feeds grid requests.
+ */
+import('lib.pkp.classes.controllers.grid.GridHandler');
+import('plugins.generic.externalFeed.controllers.grid.ExternalFeedGridRow');
+import('plugins.generic.externalFeed.controllers.grid.ExternalFeedGridCellProvider');
+
+class ExternalFeedGridHandler extends GridHandler {
+	/** @var ExternalFeedPlugin The external feeds plugin */
+	static $plugin;

```

Comment:
this is static because the LoadComponentHandler callback sets the reference to ExternalFeedPlugin through a static function call. (cf: https://github.com/kaschioudi/ojs/blob/2027-fix/plugins/generic/externalFeed/ExternalFeedPlugin.inc.php#L244)

This implementation was based on staticPages plugin
(cf: https://github.com/pkp/staticPages/blob/59aa1106de6a31210f789d3f3dda0907395a0563/StaticPagesPlugin.inc.php#L140)